### PR TITLE
fix upload_sessions error in batch updates, log every event.

### PR DESCRIPTION
**Pull request by rfornea**
-Segment logs everything.  
-Fixes a small error in actions/upload_sessions.go that was causing the data_maps array to not get populated.  There may be different ways we can approach this, if the way I have fixed it is not suitable.  
https://gophers.slack.com/archives/C3MSAFD40/p1525467579000325

### CODE REVIEW

File: actions/upload_sessions.go [hunk 1]
```diff
@@ -282,5 +282,5 @@ func (usr *UploadSessionResource) GetPaymentStatus(c buffalo.Context) error {
 }
 
 func sqlWhereForGenesisHashAndChunkIdx(genesisHash string, chunkIdx int) string {
-	return fmt.Sprintf("(genesis_hash = %s AND chunk_idx = %d)", genesisHash, chunkIdx)
+	return fmt.Sprintf("(genesis_hash = '%s' AND chunk_idx = %d)", genesisHash, chunkIdx)
```
Comment (rfornea) [inline actions/upload_sessions.go]: This was the sql error i was getting that prompted this fix:
`mysql select many: Error 1054: Unknown column '7c762a2ee5b86dc448dbed91ca75c6cede21622113f9b21e9000b4603455726a' in 'where clause'`.  
Might be some other options:  
https://gophers.slack.com/archives/C3MSAFD40/p1525467579000325

Comment (rfornea) [reply actions/upload_sessions.go]: When you log the raw query it looks like this:  
```SELECT * from data_maps WHERE (genesis_hash = c08536ccb775919edf7950a209fd5a68cc3a8c555f3209c3a89a0ecb97989ab3 AND chunk_idx = 3) OR (genesis_hash = c08536ccb775919edf7950a209fd5a68cc3a8c555f3209c3a89a0ecb97989ab3 AND chunk_idx = 2) OR (genesis_hash = c08536ccb775919edf7950a209fd5a68cc3a8c555f3209c3a89a0ecb97989ab3 AND chunk_idx = 1) OR (genesis_hash = c08536ccb775919edf7950a209fd5a68cc3a8c555f3209c3a89a0ecb97989ab3 AND chunk_idx = 0)
```

Comment (rfornea) [reply actions/upload_sessions.go]: cc @pzhao5

**Comment by pzhao5** [comment]
I should add more test for it

**Pull request merged**